### ISSUE TITLE
[FR] if(void_function())

### ISSUE THREAD

**Issue by puggan** [kalessil/phpinspectionsea#1223]
Description (screenshot):
--
When doing an if, on a function that is known to return void:
![screenshot from 2019-03-07 09-53-08](https://user-images.githubusercontent.com/866668/53944126-077bdd00-40bf-11e9-8abc-326ea350e853.png)

A warning that that if will never be true, whould be nice.

Example
--
```
if(ob_flush()) {}
```

**Comment by orklah** [comment]
This inspection is part of the default inspection by PHPStorm
PHP > Probable bugs > Void function result used > "'void' function 'ob_flush' result used"

It works with native functions, user functions and user methods alike

**Comment by puggan** [comment]
It has some wiered namespace problem

![screenshot from 2019-03-07 14-05-47](https://user-images.githubusercontent.com/866668/53958771-2b9ce580-40e2-11e9-9165-d1c05be7c050.png)
![screenshot from 2019-03-07 14-04-08](https://user-images.githubusercontent.com/866668/53958836-4d966800-40e2-11e9-9990-f7ea17f62aee.png)
![screenshot from 2019-03-07 14-06-26](https://user-images.githubusercontent.com/866668/53958787-38213e00-40e2-11e9-860c-3ae45c7ebbcc.png)

a ctrl+q says it's a void-function, but I get no warning, but if i add `\` infront of it i get the warning.

guess I should report this to PhpStorm insted then.

**Comment by puggan** [comment]
https://youtrack.jetbrains.com/issue/WI-45721
